tv_shows/get_tv_shows.py

```diff
@@ -0,0 +1,45 @@
+from requests_html import HTMLSession
+
+from tv_shows.models import TvShow
+
+session = HTMLSession()
+r = session.get('http://www.imdb.com/list/ls004729995/?sort=moviemeter,asc&st_dt=&mode=detail&page=1')
```

Comment:
Line is longer than allowed. (102 > 80)

*Origin: LineLengthBear, Section: `all.linelength`.*